src/backend/src/routes/users.routes.ts

```diff
@@ -14,5 +17,11 @@ userRouter.get('/:userId', getSingleUser);
 userRouter.get('/:userId/settings', getUserSettings);
 userRouter.post('/:userId/settings', updateUserSettings);
 userRouter.post('/auth/:login', logUserIn);
+userRouter.post(
+  '/:userId/change-role',
+  intMinZero(body('userId')),
+  isRole(body('promoteToRole')),
```

Comment:
i think instead of `promoteToRole` we should go with just `role` because it can still be a demotion as long as they're a lower rank than you